openpype/settings/defaults/project_settings/maya.json

```diff
@@ -188,18 +188,27 @@
             "whitelist_native_plugins": false,
             "authorized_plugins": []
         },
-        "ValidateCycleError": {
+        "ValidateMayaUnits": {
             "enabled": true,
             "optional": false,
-            "families": [
-                "rig"
-            ]
+            "validate_linear_units": true,
+            "linear_units": "cm",
+            "validate_angular_units": true,
+            "angular_units": "deg",
+            "validate_fps": true
         },
         "ValidateUnrealStaticMeshName": {
             "enabled": true,
             "validate_mesh": false,
             "validate_collision": true
         },
+        "ValidateCycleError": {
+            "enabled": true,
+            "optional": false,
+            "families": [
+                "rig"
+            ]
+        },
```

Comment:
I've seen this before where some keys got shuffled on "save settings". Should we enable `sort_keys=True` on the JSON dump if it currently isn't yet? Or would that produce issues with dictionary values where users might expect the same "ordering" of how they set up the dictionary when they get back to it?

It's not a big issue, but it does stand out to me that sometimes the order just shifts around a bit.

Reply:
Note: Unrelated to this PR + there will always be some issues because some are modyfing defaults manually.